src/Uno.Extensions.Templates/content/unoapp-extensions/.template.config/template.json

```
@@ -370,44 +361,44 @@
       ]
     },
     "logging": {
-      "displayName": "Extensions - Logging",
-      "description": "Includes Uno.Extensions.Logging",
+      "displayName": "Logging",
+      "description": "Configures logging support in the application",
       "type": "parameter",
       "datatype": "choice",
       "defaultValue": "serilog",
       "choices": [
         {
           "choice": "none",
-          "description": "Disables logging",
-          "displayName": "None"
+          "description": "Logging writes output to Console",
```

Much better... What do you think about this though. The logging that gets enabled in this scenario is only included for Debug builds and it's added specifically to the App Framework... so probably not something that you would use in your app for logging in general.
```suggestion
          "description": "Debug Logging for the Application Framework to the Console",
```